### PR TITLE
Add `class` or any included in `HTMLAttributes`

### PR DESCRIPTION
**Pull request by rikusen0335**
Since `<Youtube />` ocmponent doesn't accepts class, but we have a situation that we need to style the element.
I am pleasure if you could add it by this PR or anything.

Please ask me anything you don't understand, my english is so bad

### CODE REVIEW

File: packages/astro-embed-youtube/YouTube.astro [hunk 1]
```diff
@@ -2,14 +2,14 @@
 import 'lite-youtube-embed/src/lite-yt-embed.css';
 import urlMatcher from './matcher';
 
-export interface Props {
-	id: string;
+export interface Props extends astroHTML.JSX.HTMLAttributes {
+	videoIdOrUrl: string;
```
Comment (delucis) [inline packages/astro-embed-youtube/YouTube.astro:6]: I don’t think we can rename `id` at this point (maybe in the future) even though it does clash with the standard `id` attribute — bad choice on my part!

```suggestion
	id: string;
```

File: packages/astro-embed-youtube/YouTube.astro [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 	poster?: string;
 	params?: string;
 	playlabel?: string;
 }
 
-const { id, poster, ...attrs } = Astro.props as Props;
+const { videoIdOrUrl, poster, ...attrs } = Astro.props as Props;
```
Comment (delucis) [inline packages/astro-embed-youtube/YouTube.astro:12]:
```suggestion
const { id, poster, ...attrs } = Astro.props as Props;
```

File: packages/astro-embed-youtube/YouTube.astro [hunk 3]
```diff
@@ -18,7 +18,7 @@ function extractID(idOrUrl: string) {
 	return urlMatcher(idOrUrl);
 }
 
-const videoid = extractID(id);
+const videoid = extractID(videoIdOrUrl);
```
Comment (delucis) [inline packages/astro-embed-youtube/YouTube.astro:21]:
```suggestion
const videoid = extractID(id);
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks @rikusen0335! Makes sense to extend the `Props` type like this 👍 

Changing `id` to `videoIdOrUrl` will be a breaking change (as you can see from the failing tests) and while sympathetic, I’d like to keep it as is for now.

**Comment by rikusen0335** [comment]
Ohhhh yes, I forgot about backward compatibilities. We should stay keep id's not changed.

**Comment by rikusen0335** [comment]
I commited your change requests.
Let's see if the build work

**Review by delucis — APPROVED** [review]
I need to update some dependencies to make the tests run but I'm confident in this change nonetheless.

Thanks!

**Pull request merged**